angular.json

```diff
@@ -43,14 +43,13 @@
             "tsConfig": "tsconfig.app.json",
             "aot": true,
             "assets": ["src/favicon.ico", "src/assets"],
-            "styles": ["node_modules/@clr/icons/clr-icons.min.css", "node_modules/@clr/ui/clr-ui.min.css", "src/styles.scss"],
+            "styles": ["node_modules/bootstrap/scss/bootstrap.scss", "src/styles.scss"],
```

Comment:
Can we use [functions](https://getbootstrap.com/docs/4.0/getting-started/theming/#functions) when importing the scss like this, or do we need to import it in our own stylesheet?

Reply:
Good question, I only tested if styling worked as expected. I have little experience with SCSS so this is hard for me to test, could you or @RensNooren perhaps test this?